src/components/MemoryLeak/index.jsx

```diff
@@ -7,27 +7,21 @@ import grateUtil from 'src/utils/grade';
 import Constants from 'src/constants';
 import RichText from 'src/components/customize/RichText';
 
-const { generalMarkMap, getMemoryLeakMark } = grateUtil
+const { generalMarkMap, getMemoryLeakMark, getMemoryLeakDataSummaryDescription } = grateUtil
 
 const MemoryLeak = () => {
   const performanceData = useContext(Context);
   if (!performanceData) {
-    return null;
+    // return null;
   }
   const { memoryLeakData } = performanceData;
   if (!memoryLeakData || memoryLeakData.length < 1) {
-    return null;
+    // return null;
   }
```

Comment:
when no data, still have to render this compoent, these code can be removed

Reply:
removed